core/state_processor_test.go

```diff
@@ -300,6 +301,71 @@ func TestStateProcessorErrors(t *testing.T) {
 			}
 		}
 	}
+
+	{
```

Comment:
Can we move this to a separate test function?

Reply:
done